### DISCUSSION TITLE
Disable /fly when in combat

### DISCUSSION THREAD

**Discussion by hotspuraz** [Zrips/CMI#3792] [Feature Requests]
**Description of feature:**
If a player is attacked or attacks another player, disable their fly and stop them from using the command again for X seconds.

---
### Basic explanation as to why this feature is needed and/or useful to have: ###

I have run a server for a long time that offers /fly at a certain vote rank but as there is a requirement to get to this rank, not everyone has it. Therefore, I have always kept PVP off. It would be nice to be able to disable /fly in combat so we can have fair pvp on the server :)

**Comment by CloudeLecaw** [comment]
Why cmi should do this? Cmi is no PvPMAnagement luke the plugin PvPManager that handles things like this.

There are 10000 option that could be dine while players are in pvp or to prevent unfair pvp.
So i suggest using a plugin like PvP-Manager there isba free and Premium Version of this plugin allready.  Well i know tomirrow someone wants the Feature to have zone created anr manged with cmi, or multiple inventories, just a question of time...

**Comment by hotspuraz** [comment]
It's a fair point, CMI does a lot already x)
